e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -116,10 +118,11 @@ suite('Language server validation', async () => {
         await editor.waitSuggestion(javaFileName, 'run(Class<?> primarySource, String... args) : ConfigurableApplicationContext');
     });
 
-    // it's skipped because of issue https://github.com/eclipse/che/issues/14520
-    test.skip('Codenavigation', async () => {
+
+    test('Codenavigation', async () => {
         await editor.moveCursorToLineAndChar(javaFileName, 32, 17);
-        await editor.performKeyCombination(javaFileName, Key.chord(Key.CONTROL, Key.F12));
+        await checkCodeNavigationWithContextMenu();
+        // await editor.performKeyCombination(javaFileName, Key.chord(Key.CONTROL, Key.F12));
```

Comment:
What is a purpose of this comment?

Reply:
replaced to try/catch block with log error